Header header

string name
string status
string mode 
uint32 connected_clovers 
uint32 armed_clovers
